src/client/swagger/managedFunctions.yml

```diff
@@ -167,57 +167,94 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/api/v2private/functions/{functionID}/runs':
+  '/api/v2/functions/{functionID}/invoke':
     get:
-      operationId: GetFunctionsIDRuns
+      operationId: GetFunctionsIDInvoke
       tags:
         - Functions
-      summary: List runs for a function
+      summary: Manually invoke a function with params in query
       parameters:
         - in: path
           name: functionID
           schema:
             type: string
           required: true
-          description: The ID of the function to get runs for.
+        - in: query
+          name: params
+          schema:
+            type: object
+            additionalProperties: true
+          style: form
+          explode: true
       responses:
         '200':
```

Comment:
I think this should be `default` instead remove the default below. We also don't know if the user won't return a non-error response with specific code - like 20x responses (like 201, 204) or 30x redirects.

Reply:
fixed.